app/lib/efile/nj/nj_senior.rb

```diff
@@ -0,0 +1,11 @@
+module Efile
+  module Nj
+    class NjSenior
+      def self.is_over_65(birth_date)
```

Comment:
[pebble] The base intake model already has methods `primary_senior?` and `spouse_senior?`

Reply:
i moved the age check logic (and wages logic) out of the calculator since it now needs to be used in two separate places-- let me know if that choice sounds ok

Reply:
oh i think i may have misread your comment -- i simply was just taking our existing logic that explicitly compares birth year... so moving to a `primary_senior?` would be a refactor to our existing code. i'm ok with that but just noting that it would effect beyond this PR, so I'd be relying on existing tests to make sure that refactor is ok. I'll try it though, since it would be nice to just not have this util at all...

Reply:
great catch - using those methods does indeed still pass tests, so i will refactor the original in the calculator to just use that method too